### DISCUSSION TITLE
cron schedule ignore date range.

### DISCUSSION THREAD

**Discussion by 63110** [community/community#107534] [Actions]
### Select Topic Area

Bug

### Body

I have the next pipeline with cron

name: test

on:
  schedule:
    - cron: '30 2 1-7 * 6'
    
 
 Without any reason was triggered on Saturday but on 10th February. Can someone explain why? Looks like a bug if was set 1-7

**Comment by ZephyrusBlaze** [comment]


**Reply by 63110** [reply]
Crontab guru, provided another information 

“At 02:30 on every day-of-month from 1 through 7 and on Saturday.”

**Comment by 63110** [comment]
Changed to this 
30 2 1-7 * *
At 02:30 AM, between day 1 and 7 of the month
